### PR TITLE
FreewheelSSP bidder updates

### PR DESCRIPTION
**Pull request by KolomiietsSerhii**
Added support of GPP, ZoneID support of Integer or String values.

### CODE REVIEW

**Comment by AntoxaAntoxic** [comment]
@SerhiiNahornyi @KolomiietsSerhii This is the bidder that can be added as the alias for the generic bidder, isn't it?

**Review by And1sS — COMMENTED** [review]
In go version of bid adapter they also add this header
```go
headers.Add("Componentid", "prebid-go")
```
Which looks like an indicator that request comes from PBS-Go.

I guess we should check whether PBS-Java has to add similar indicator that request is coming from PBS-Java.

File: src/main/java/org/prebid/server/bidder/freewheelssp/FreewheelSSPBidder.java [hunk 1]
```diff
@@ -35,11 +36,13 @@ public FreewheelSSPBidder(String endpointUrl, JacksonMapper mapper) {
 
     @Override
     public final Result<List<HttpRequest<BidRequest>>> makeHttpRequests(BidRequest bidRequest) {
+        final MultiMap headers = HttpUtil.headers();
+        headers.add("Componentid", "prebid-java");
```
Comment (SerhiiNahornyi) [inline src/main/java/org/prebid/server/bidder/freewheelssp/FreewheelSSPBidder.java:40]: Pls create constant of `"prebid-java"`

Comment (SerhiiNahornyi) [inline src/main/java/org/prebid/server/bidder/freewheelssp/FreewheelSSPBidder.java:40]: Pld add test case for this change

**Review by SerhiiNahornyi — CHANGES_REQUESTED** [review]

File: src/main/java/org/prebid/server/bidder/freewheelssp/FreewheelSSPBidder.java (hunk 1 above)
Comment (KolomiietsSerhii) [reply src/main/java/org/prebid/server/bidder/freewheelssp/FreewheelSSPBidder.java]: Done.

**Review by SerhiiNahornyi — APPROVED** [review]

**Pull request merged**